src/chapters/css/rules.rst

```diff
@@ -0,0 +1,67 @@
+CSS Rules
+=========
+
+How to Write a Style Rules
+--------------------------
+
+CSS style rules cover an infinite amount of amount of possibilities.
+Programmers initialize individual CSS properties within the declaration block.
+Every property in CSS has a default value. For example, ``font-color`` defaults to "black".
+Programmers only have to change what needs to be.
```

Comment:
"Programmers only have to change what needs to be." Can this be said differently or possibly removed? I feel it may be unnecessary.